examples/ecs/provisioners/README.md

```diff
@@ -1,4 +1,75 @@
-# Using Provisioner over SSH to an ECS Instance
+# Using Provisioner over SSH to an ECS instance
 
-This example provisions an ECS instance with a Public IP Address
-and [runs a `remote-exec` provisioner](https://www.terraform.io/docs/provisioners/remote-exec.html) over SSH.
+This example provisions an ECS instance with a Public IP address and runs a `remote-exec` provisioner over SSH.
+
+## Using private key to remote ECS instance
+
+For `huaweicloud_compute_instance` resoruce, if If the user_data field is used, the `admin_pass` field will be invalid.
```

Comment:
s/if If/if

Reply:
if `user_data` specified, `admin_pass` setting will not take effect.

Reply:
The description has been updated.